### PR TITLE
tgui: Jukebox

### PR DESCRIPTION
**Pull request by Arkatos1**
## About The Pull Request

This PR converts Jukebox from a html based interface to tgui, with some tweaks and new additions. A new feature was added to a jukebox - you can now set a volume for a songs played, instead of them always playing at a max volume. Useful if a song is very loud or annoying. Comparison images below.

**OLD:**

JukeboxOLD

**NEW:**

JukeboxNEW

## Why It's Good For The Game

Better UI and UX.

## Changelog
:cl: Arkatos
add: Jukebox now uses tgui.
tweak: You can now set a volume of a jukebox songs.
/:cl:

### CODE REVIEW

**Comment by Fikou** [comment]
is volume for the client or everyone?

**Comment by Arkatos1** [comment]
Everyone.

File: tgui/packages/tgui/interfaces/Jukebox.js [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+              {track_selected ? track_length : "No Track Selected"}
+            </LabeledList.Item>
+            <LabeledList.Item label="Track Beat">
+              {track_selected ? track_beat : "No Track Selected"}
+              {track_beat === 1 ? " beat" : " beats"}
+            </LabeledList.Item>
+          </LabeledList>
+        </Section>
+        <Section title="Machine Settings">
+          <LabeledList>
```
Comment (stylemistake) [inline tgui/packages/tgui/interfaces/Jukebox.js:54]: Should be LabeledControls

Comment (Arkatos1) [reply tgui/packages/tgui/interfaces/Jukebox.js]: Fixed.

**Comment by Learner10** [comment]
> Everyone.

IB4 emagging a jukebox locks the volume setting at 200% and queue's up CaramellDansen causing physical discomfort and harm to the players behind their characters.

**Pull request merged**